### PR TITLE
[docker] migrate otbr docker to OpenThread daemon mode

### PR DESCRIPTION
**Pull request by gjc13**

### CODE REVIEW

File: script/_otbr [hunk 1]
```diff
@@ -50,17 +54,21 @@ otbr_install()
...
     sudo make install
     if have systemctl; then
         sudo systemctl reload dbus
         sudo systemctl daemon-reload
-        sudo systemctl enable wpantund otbr-web otbr-agent || true
-        sudo systemctl is-enabled wpantund || die 'Failed to enable wpantund!'
+        sudo systemctl enable otbr-web otbr-agent || true
         sudo systemctl is-enabled otbr-agent || die 'Failed to enable otbr-agent!'
         sudo systemctl is-enabled otbr-web || die 'Failed to enable otbr-web!'
+        with OTBR_DAEMON || {
```
Comment (bukepo) [inline script/_otbr:68]: Should this be started before otbr-agent and otbr-web?

File: etc/docker/docker_entrypoint.sh [hunk 2]
```diff
@@ -84,12 +83,7 @@ NAT64_PREFIX=${NAT64_PREFIX/\//\\\/}
 sed -i "s/^prefix.*$/prefix $NAT64_PREFIX/" /etc/tayga.conf
 sed -i "s/dns64.*$/dns64 $NAT64_PREFIX {};/" /etc/bind/named.conf.options
 
-echo "Config:NCP:SocketPath \"$NCP_PATH\"" > /etc/wpantund.conf
-echo "Config:TUN:InterfaceName $TUN_INTERFACE_NAME " >> /etc/wpantund.conf
-echo "Daemon:SetDefaultRouteForAutoAddedPrefix $AUTO_PREFIX_ROUTE" >> /etc/wpantund.conf
-echo "IPv6:SetSLAACForAutoAddedPrefix $AUTO_PREFIX_SLAAC" >> /etc/wpantund.conf
-
-echo "OTBR_AGENT_OPTS=\"-I $TUN_INTERFACE_NAME\"" > /etc/default/otbr-agent
+echo "OTBR_AGENT_OPTS=\"-I $TUN_INTERFACE_NAME $NCP_PATH 115200\"" > /etc/default/otbr-agent
```
Comment (bukepo) [inline etc/docker/docker_entrypoint.sh:86]:
```suggestion
echo "OTBR_AGENT_OPTS=\"-I '$TUN_INTERFACE_NAME' '$NCP_PATH' 115200\"" > /etc/default/otbr-agent
```

Comment (gjc13) [reply etc/docker/docker_entrypoint.sh:86]: That will break the service init as it will then treat `$NCP_PATH` for a separate command.

**Comment by jwhui** [comment]
Should we look to update the OTBR Docker guide on openthread.io to be consistent with the change to daemon mode?

https://openthread.io/guides/border-router/docker

**Comment by gjc13** [comment]
> Should we look to update the OTBR Docker guide on openthread.io to be consistent with the change to daemon mode?
> 
> https://openthread.io/guides/border-router/docker

I've chatted with Jeff and he has pointed where to update the docs.

**Comment by gjc13** [comment]
No longer required.

**Pull request closed without merging**